### PR TITLE
Generate Ruby DLP

### PR DESCRIPTION
**Pull request by geigerj**

### CODE REVIEW

**Comment by geigerj** [comment]
@swcloud @neozwu ping

@lukesneeringer any idea why CI is not running?

**Comment by lukesneeringer** [comment]
> @lukesneeringer any idea why CI is not running?

I turned off building on forks by accident. I turned it back on this morning.

**Comment by geigerj** [comment]
@swcloud regenerated.

File: generated/ruby/google-cloud-ruby/google-cloud-dlp/google-cloud-privacy-dlp.gemspec [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |gem|
+  gem.name          = "google-cloud-privacy-dlp"
+  gem.version       = "0.6.8"
```
Comment (swcloud) [inline generated/ruby/google-cloud-ruby/google-cloud-dlp/google-cloud-privacy-dlp.gemspec]: I think we can configure the generated package version for ruby in artman_dlp.yaml just like

https://github.com/googleapis/googleapis/blob/master/gapic/api/artman_dlp.yaml#L48

Similar to video intelligence, the start version can be '0.20.0'.

Comment (geigerj) [inline generated/ruby/google-cloud-ruby/google-cloud-dlp/google-cloud-privacy-dlp.gemspec]: Done.

**Pull request merged**